Quadrat/update font sizes

jeffikus opened the pull request:
#### Changes proposed in this Pull Request:

Updates the font sizes to match the Design file changes from @beafialho 

#### Related issue(s):

None.

#### Test Plan:

**Test that the font sizes on Desktop are:**
h1: 80px
h2: 65px
h3: 38px
h4: 22px
h5: 20px
h6: 18px
P Tiny: 16px
P Small: 18px
P Normal: 20px
P Large: 22px
P Extra Large: 38px
P Huge: 48px
Site Title: 20px
Site Tag: 16px

**Test that the font sizes on Mobile are:**
h1: 48px
h2: 36px
h3: 28px
h4: 22px
h5: 20px
h6: 18px
P Tiny: 16px
P Small: 18px
P Normal: 20px
P Large: 22px
P Extra Large: 28px
P Huge: 36px

Test that Quote Citation font size is: 16px
Test that Query Pagination font size is: 20px

jeffikus committed:
Updates themes.json htag font sizes

Updates font sizes for h4,5,6 in theme.json.

jeffikus committed:
Updates typography font sizes.

Updates tiny, small, normal, and extra large font sizes.

jeffikus committed:
Updates font sizes.

Citation and query pagination block font size updates.

jeffikus committed:
Updates font sizes.

Navigation and site title block font size update.

jeffikus committed:
Navigation font size update.

Updates responsive navigation font size.

scruffian commented on quadrat/child-theme.json:
```diff
@@ -278,7 +278,7 @@
 			"core/navigation": {
 				"typography": {
-					"fontSize": "18px"
+					"fontSize": "20px"
```
This should use the variable `var(--wp--preset--font-size--normal)`

jeffikus replied:
Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b

scruffian commented on quadrat/child-theme.json:
```diff
@@ -306,7 +306,7 @@
 				"citation": {
 					"typography": {
-						"fontSize": "15px",
+						"fontSize": "16px",
```
This should use the variable var(--wp--preset--font-size--tiny)

jeffikus replied:
Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b

scruffian commented on quadrat/child-theme.json:
```diff
@@ -349,7 +349,7 @@
 			"core/query-pagination": {
 				"typography": {
-					"fontSize": "18px",
+					"fontSize": "20px",
```
This should use the variable var(--wp--preset--font-size--normal)

jeffikus replied:
Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b

scruffian commented on quadrat/child-theme.json:
```diff
@@ -358,7 +358,7 @@
 				},
 				"typography": {
-					"fontSize": "18px",
+					"fontSize": "20px",
```
This should use the variable `var(--wp--preset--font-size--normal)`

jeffikus replied:
Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b

scruffian commented on quadrat/child-theme.json:
```diff
@@ -389,21 +389,21 @@
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--dm-sans)",
-					"fontSize": "20px",
+					"fontSize": "22px",
```
This should use the variable `var(--wp--preset--font-size--large)`

jeffikus replied:
Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b

scruffian commented on quadrat/child-theme.json:
```diff
@@ -389,21 +389,21 @@
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--dm-sans)",
-					"fontSize": "18px",
+					"fontSize": "20px",
```
This should use the variable `var(--wp--preset--font-size--normal)`

jeffikus replied:
Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b

scruffian commented on quadrat/child-theme.json:
```diff
@@ -389,21 +389,21 @@
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--dm-sans)",
-					"fontSize": "16px",
+					"fontSize": "18px",
```
This should use the variable `var(--wp--preset--font-size--small)`

jeffikus replied:
Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b

scruffian commented on quadrat/sass/blocks/_navigation.scss:
```diff
@@ -45,7 +45,7 @@
 						.wp-block-navigation-link__content {
 							padding: 0;
-							font-size: 15px;
+							font-size: 16px;
```
This should use the variable `var(--wp--preset--font-size--tiny)`

jeffikus replied:
Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b

jeffikus committed:
Fixes from review.

Changes font sizes to variables.

Props @scruffian for review.

scruffian commented on quadrat/sass/blocks/_navigation.scss:
```diff
@@ -45,7 +45,7 @@
 						.wp-block-navigation-link__content {
 							padding: 0;
-							font-size: 15px;
+							font-size: var(--wp--preset--font-size--tiny);
```
There's a font size declaration in line 31 of this file too..

jeffikus replied:
Fixed in https://github.com/Automattic/themes/pull/4683/commits/2532bae61948e88efce92f0cb2b3f7adf7d75527 and did a sweep in the theme for other areas of font-size in https://github.com/Automattic/themes/pull/4683/commits/51266f3eea9ac012a6606ccf601ef253d6f68976

jeffikus committed:
Replace font-size with variable.

Well spotted @scruffian - updated this and did a sweep for any other font-size in the theme. I don't see any more.

jeffikus committed:
Another sweep of fontSize replacement.

Found another instance in custom typography and core/code block.

scruffian commented:
This is looking good. We need to make the same changes to geologist.

jeffikus commented:
Yes, I'll get cracking on the changes to Geologist later today as well. Thanks for the review!

scruffian commented:
Deployed in D67302-code
